fastplotlib/widgets/histogram_lut.py

```diff
@@ -178,15 +180,17 @@ def _calculate_histogram(self, data):
         )
 
         edges_flanked = np.concatenate((flank_left, edges, flank_right))
-        np.unique(np.diff(edges_flanked))
 
         hist_flanked = np.concatenate(
             (np.zeros(flank_nbins), hist, np.zeros(flank_nbins))
         )
 
         # scale 0-100 to make it easier to see
         # float32 data can produce unnecessarily high values
-        hist_scaled = hist_flanked / (hist_flanked.max() / 100)
+        hist_scale_value = hist_flanked.max()
+        if np.allclose(hist_scale_value, 0):
+            hist_scale_value = 1
+        hist_scaled = hist_flanked / (hist_scale_value / 100)
```

Comment:
ah so then we just get a flat histogram, all bins have zero weight?

Reply:
@kushalkolar Yeah, this is really just an extra check in case an empty array is passed in (in which case the histogram weights will be 0.